vjezba10_11/vjezba10_11/city_tree.c

```diff
@@ -59,22 +58,22 @@ CityTreeP ReadCitiesFromFile(CityTreeP root, char* citiesFileName)
     FILE* fp = NULL;
     char cityName[MAX_SIZE] = { 0 };
     int population = 0;
+    int i;
 
     fp = fopen(citiesFileName, "r");
-    if (fp == NULL)
+    if (NULL == fp)
     {
-        perror("File can't be opened!\n");
+        perror("\nFile can't be opened or doesn't exist yet!\n");
         return;
     }
 
     int numOfLines = 0;
     char buffer[MAX_SIZE];
     while (fgets(buffer, MAX_SIZE, fp))
-    {
         numOfLines++;
-    }
+    
     rewind(fp);
-    for (int i = 0; i < numOfLines; i++)
```

Comment:
*ponovljeno*, sve varijable deklarirati izvan
  samog uvjeta petlje , zbog starijih verzija compiler-a

  NE: for(int i = 0; i < n; i++ )
  DA:
     {
	int i;
        .
	.
	.
	for(i = 0; i < n; i++ )
    		...
     }